extensions/vscode-api-tests/src/singlefolder-tests/terminal.test.ts

```diff
@@ -850,17 +850,17 @@ import { assertNoRpc, poll } from '../utils';
 				collection.prepend('C', '~c2~');
 
 				// Verify get
-				deepStrictEqual(collection.get('A'), { value: '~a2~', type: EnvironmentVariableMutatorType.Replace });
-				deepStrictEqual(collection.get('B'), { value: '~b2~', type: EnvironmentVariableMutatorType.Append });
-				deepStrictEqual(collection.get('C'), { value: '~c2~', type: EnvironmentVariableMutatorType.Prepend });
+				deepStrictEqual(collection.get('A'), { value: '~a2~', type: EnvironmentVariableMutatorType.Replace, scope: undefined });
```

Comment:
```suggestion
				deepStrictEqual(collection.get('A'), { value: '~a2~', type: EnvironmentVariableMutatorType.Replace });
```

Reply:
IMO, it would look cleaner if you removed all of the `scope: undefined`

Reply:
We can make it optional when using internally. For external use though, keeping it as Required: https://github.com/microsoft/vscode/pull/179323#discussion_r1172927470.